Formula/ammonite-repl.rb

```diff
@@ -1,10 +1,11 @@
 class AmmoniteRepl < Formula
   desc "Ammonite is a cleanroom re-implementation of the Scala REPL"
   homepage "https://ammonite.io/"
-  url "https://github.com/lihaoyi/Ammonite/releases/download/2.4.0/3.0-2.4.0"
+  url "https://github.com/lihaoyi/Ammonite/releases/download/2.4.0/2.13-2.4.0"
```

Comment:
Can we add a comment here that explains which version of Ammonite to use? i.e. between the `3.0-x` or `2.13-x`, etc versions.

Reply:
I've added some notes now